templates/contrib/jdbc/legacy/build.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
```

Comment:
I know this script is meant to be used internally only. But with a few changes, we could make it more general purpose to cope with Openshift and any docker registry:
- Avoid doing the docker login here (the main issue is that the docker registry might not be the one in Openshift). See more about this above.
- Make the docker registry namespace configurable
- Adjust create_build method to parametrize the Openshift namespace (at the moment, it's openshift).

See [this commit](https://github.com/jboss-container-images/rhpam-7-openshift-image/commit/0be5030b07c4f1b676e615e47ce52206826a4dcc) as a reference.